src/qml/FeatureListForm.qml

```diff
@@ -572,16 +572,19 @@ Rectangle {
 
   function hide()
   {
-    props.isVisible = false
-    focus = false
-    fullScreenView = qfieldSettings.fullScreenIdentifyView
+    props.isVisible = false;
```

Comment:
Do we move to semicolon syntax? I am pretty sure most of the codebase is without?

Reply:
I don't know if we have an agreement on this :) on my end, I try to add whenever I stumble on semicolon-less { functions } (but not for one-liners).